composio/tools/__init__.py

```diff
@@ -53,6 +53,14 @@ def __init__(
         )
         self.runtime = runtime
```

Comment:
How was this working earlier?

Reply:
I did not see it to be used anywhere...it was just being set, so I just added proper setter, cause when we inherit from a child class, like from langchain for llamaindex, it was not possible to set that externally.